test/ecto/adapters/postgres_test.exs

```diff
@@ -56,7 +56,7 @@ defmodule Ecto.Adapters.PostgresTest do
   defp execute_ddl(query), do: query |> SQL.execute_ddl |> Enum.map(&IO.iodata_to_binary/1)
 
   defp insert(prefx, table, header, rows, on_conflict, returning) do
-    IO.iodata_to_binary SQL.insert(prefx, table, header, rows, on_conflict, returning)
+    IO.iodata_to_binary SQL.insert(prefx, table, header, rows, on_conflict, returning, [])
```

Comment:
Let's also add that checks the proper references are used and counted correctly for placeholders!

Reply:
Both in postgres and tds suites!